Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -1406,7 +1406,9 @@ protected class NullClassWriter : IClassWriter {
       public ConcreteSyntaxTree SynthesizeMethod(Method m, List<TypeArgumentInstantiation> typeArgs, bool createBody, bool forBodyInheritance, bool lookasideBody) {
         throw new UnsupportedFeatureException(m.tok, Feature.MethodSynthesis);
       }
-
+      public ConcreteSyntaxTree CreateFreshMethod(Method m) {
```

Comment:
We don't need this method any more, correct?

Reply:
Correct!